cmd/manager/main.go

```diff
@@ -149,16 +147,19 @@ func main() {
 		WithServiceName(operatorconfig.OperatorName).
 		GetConfig()
 
-	// detect if operator-sdk up local is run
-	detectSDKLocal := os.Getenv(envSDK)
-
-	// Configure metrics. If it errors, log the error and exit.
-	if detectSDKLocal == "local" {
-		log.Info("Skipping metrics configuration; not running in a cluster.")
+	// Get the namespace the operator is currently deployed in.
+	if _, err := k8sutil.GetOperatorNamespace(); err != nil {
+		if errors.Is(err, k8sutil.ErrRunLocal) {
+			log.Info("Skipping CR metrics server creation; not running in a cluster.")
+		} else {
+			log.Error(err, "Failed to get operator namespace")
+		}
 	} else {
 		if err := metrics.ConfigureMetrics(context.TODO(), *metricsServer); err != nil {
 			log.Error(err, "Failed to configure Metrics")
 			os.Exit(1)
+		} else {
```

Comment:
I think this could just be
```
if err := metrics.ConfigureMetrics(context.TODO(), *metricsServer); err != nil {
        log.Error(err, "Failed to configure Metrics")
        os.Exit(1)
}
log.Info("Successfully configured Metrics")
```
Just saves a line of code